### PR TITLE
Updated role for standalone installation (now can configure nginx)

### PR DESCRIPTION
**Pull request by digitalvintage**
Hi!
I've updated this role to provide unattended deploy of Wok/Kimchi on a server.
Now it can control nginx (it is recommended for server to have only Kimchi role).

### CODE REVIEW

File: tasks/config.yml [hunk 1]
```diff
@@ -15,6 +15,45 @@
...
+    backup: yes
+  notify: restart nginx
+
+- name: configure nginx for Wok server
+  template:
+    src: nginx_nossl_wok.conf.j2
+    dest: /etc/nginx/conf.d/wok.conf
+    force: yes
+    backup: yes
+  when: (wok.https_redirect == true) and (wok.nginx_ssl_disable == false)
```
Comment (bliemli) [inline tasks/config.yml]:
```suggestion
  when:
  - wok.https_redirect
  - not wok.nginx_ssl_disable
```

File: tasks/config.yml [hunk 2]
```diff
@@ -15,6 +15,45 @@
...
+  notify: restart nginx
+
+- name: configure nginx for Wok server
+  template:
+    src: nginx_nossl_wok.conf.j2
+    dest: /etc/nginx/conf.d/wok.conf
+    force: yes
+    backup: yes
+  when: (wok.https_redirect == false) and (wok.nginx_ssl_disable == false)
```
Comment (bliemli) [inline tasks/config.yml]:
```suggestion
  when:
  - not wok.https_redirect
  - not wok.nginx_ssl_disable
```

**Comment by digitalvintage** [comment]
I have accepted changes

**Comment by bliemli** [comment]
Thank you for your contribution!

**Pull request merged**